### PR TITLE
NOTIF-100 Add notifications to user-preferences

### PR DESCRIPTION
**Pull request by josejulio**
Depends on https://github.com/RedHatInsights/notifications-backend/pull/95

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -26,6 +26,18 @@
...
         },
+        "notifications": {
+            "title": "Notifications (Beta)",
+            "permissions": [
+                {
+                    "method": "hasPermissions",
+                    "args": [["notifications:*:*"]]
+                },
+                {
+                    "method": "isBeta"
```
Comment (josejulio) [inline src/config.json:37]: @karelhala @ryelo :
Is possible to negate isBeta?

I would like to know if is possible to. hide policies settings when using stable.

Comment (karelhala) [reply src/config.json:37]: That's exactly what the `isBeta` does, it will be visible only for beta environments. Hidden on any stable ones.

**Review by karelhala — APPROVED** [review]
Looking good!

File: src/config.json (hunk 1 above)
Comment (josejulio) [reply src/config.json:37]: Sorry, I worded it the other way. I want to hide it in `beta` in the end.

Is still unclear if we want to do that, but in `beta` we want to replace our Policies email preferences with Notifications.

So we have Policies and Notifications and something like this is being discussed:

Beta:
 Only shows Notifications

Stable:
 Only shows Policies

Comment (karelhala) [reply src/config.json:37]: Oh, okay. There is no check for that thing, but you can write it you want and enhance this constant https://github.com/RedHatInsights/user-preferences-frontend/blob/master/src/Utilities/functions.js#L10

**Pull request merged**